src/SAMLTrace.js

```diff
@@ -280,6 +280,20 @@ SAMLTrace.Request.prototype = {
       let isInPost = isAnyParameterInCollection(parameters, this.post);
       return isInGet || isInPost;
     };
+    
+    const isSamlResponse = () => {
+      const parameters = ["SAMLResponse"];
+      let isInGet = isAnyParameterInCollection(parameters, this.get);
+      let isInPost = isAnyParameterInCollection(parameters, this.post);
+      return isInGet || isInPost;
+    };
+    
+    const isSamlRequest = () => {
+      const parameters = ["SAMLRequest"];
+      let isInGet = isAnyParameterInCollection(parameters, this.get);
+      let isInPost = isAnyParameterInCollection(parameters, this.post);
+      return isInGet || isInPost;
+    };    
```

Comment:
These two functions can now be omitted.

Reply:
> Could it be you accidently dropped some keys?

It wasn't accidental drop. Just wanted to confirm if this convention will be acceptable so I didn't rewrote all cases. 

As for multiple-value attribute representation I agree that in case of little bit longer values we would have word wrapping. However it will be good to differentiate this somehow from single-value attributes. Maybe like here:
![screenshot2](https://user-images.githubusercontent.com/29899901/61176125-9e9bf880-a5bb-11e9-8a10-1d5b50f0562f.png)

But we are starting to play in asciiart that way :)

Reply:
Please don't forget to delete `isSamlResponse` and `isSamlRequest` since they aren't needed anymore 😉